public/diagnostic.js

```diff
@@ -23,3 +23,36 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Cart = Ember.Object.extend({
+  listOfOrders: [],
+  addToCart : Ember.computed('orders', function() {
+    let orders = Order.get('orders');
+    listOfOrders.pushObject(orders);
+  })
+  totalPrice : Ember.computed('orders.@each.price', function(){
+    let orders = this.get('orders');
+    let sum = orders.map((order) => order.price).reduce((a,b) => a + b);
+    if (orders.length > 0) {
+      return sum;
+    } else {
+      return 0;
+    }
+  })
+});
+
+const Order = Cart.extend({
+  orderPrice : Ember.computed('orders.@each.price', function(){
```

Comment:
`orderPrice` should be computed based on an individual order's `quantity` and `price` properties.